SCClassLibrary/Common/Control/MIDIOut.sc

```diff
@@ -94,6 +94,16 @@ MIDIClient {
 	*externalDestinations{
 		^destinations;
 	}
+
+	*getClientID {
```

Comment:
semantically fine with me, but would be an API change. 
don't know if it ever occurs in user code on linux - does it?